packages/wp-story-editor/src/components/metaBoxes/metaBoxesProvider.js

```diff
@@ -44,21 +48,27 @@ function MetaBoxesProvider({ children }) {
   const state = useMemo(
     () => ({
       state: {
+        isSavingMetaBoxes,
```

Comment:
Where are `isSavingMetaBoxes` and `setIsSavingMetaBoxes` actually used? `useSaveMetBoxes` has its own state for that?

Reply:
`useSaveMetBoxes` state definition was left in doing back and forth of code. Removed from `useSaveMetBoxes` in https://github.com/google/web-stories-wp/pull/9027/commits/cde6cce47015e688b6f722aac5b4533016a5e84e 
Thank you